mycroft/audio/speech.py

```diff
@@ -118,23 +119,30 @@ def mute_and_speak(utterance, ident):
         tts = TTSFactory.create()
         tts.init(bus)
         tts_hash = hash(str(config.get('tts', '')))
+        tts.set_fall_back(False)
 
     LOG.info("Speak: " + utterance)
     try:
         tts.execute(utterance, ident)
     except RemoteTTSTimeoutException as e:
         LOG.error(e)
+        # Clear the previous cache
+        tts.clear_cache()
```

Comment:
Is the subfolder needed if the cache is cleared

Reply:
Yeah this is not mandatory, just to clear the old data